Library/Formula/xorg.rb

```diff
@@ -0,0 +1,470 @@
+    sha256   "4c675533e79cd730997d232c8894b6692174dce58d3e207021b8f860be498468"
+  end
+
+  resource "recordproto" do
+    url      "http://ftp.x.org/pub/individual/proto/recordproto-1.14.2.tar.bz2"
+    sha256   "a777548d2e92aa259f1528de3c4a36d15e07a4650d0976573a8e2ff5437e7370"
+  end
+
+  resource "renderproto" do
+    url      "http://ftp.x.org/pub/individual/proto/renderproto-0.11.1.tar.bz2"
+    sha256   "06735a5b92b20759204e4751ecd6064a2ad8a6246bb65b3078b862a00def2537"
+  end
+
+  resource "resourceproto" do
+    url      "http://ftp.x.org/pub/individual/proto/resourceproto-1.2.0.tar.bz2"
+    sha256   "3c66003a6bdeb0f70932a9ed3cf57cc554234154378d301e0c5cfa189d8f6818"
+  end
+
+  resource "scrnsaverproto" do
+    url      "http://ftp.x.org/pub/individual/proto/scrnsaverproto-1.2.2.tar.bz2"
+    sha256   "8bb70a8da164930cceaeb4c74180291660533ad3cc45377b30a795d1b85bcd65"
+  end
+
+  resource "videoproto" do
+    url      "http://ftp.x.org/pub/individual/proto/videoproto-2.3.2.tar.bz2"
+    sha256   "8dae168cb820fcd32f564879afb3f24d27c176300d9af66819a18265539bd4b6"
+  end
+
+  resource "xcmiscproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xcmiscproto-1.2.2.tar.bz2"
+    sha256   "b13236869372256c36db79ae39d54214172677fb79e9cdc555dceec80bd9d2df"
+  end
+
+  resource "xextproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xextproto-7.3.0.tar.bz2"
+    sha256   "f3f4b23ac8db9c3a9e0d8edb591713f3d70ef9c3b175970dd8823dfc92aa5bb0"
+  end
+
+  resource "xf86bigfontproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xf86bigfontproto-1.2.0.tar.bz2"
+    sha256   "ba9220e2c4475f5ed2ddaa7287426b30089e4d29bd58d35fad57ba5ea43e1648"
+  end
+
+  resource "xf86dgaproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xf86dgaproto-2.1.tar.bz2"
+    sha256   "ac5ef65108e1f2146286e53080975683dae49fc94680042e04bd1e2010e99050"
+  end
+
+  resource "xf86driproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xf86driproto-2.1.1.tar.bz2"
+    sha256   "9c4b8d7221cb6dc4309269ccc008a22753698ae9245a398a59df35f1404d661f"
+  end
+
+  resource "xf86vidmodeproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xf86vidmodeproto-2.3.1.tar.bz2"
+    sha256   "45d9499aa7b73203fd6b3505b0259624afed5c16b941bd04fcf123e5de698770"
+  end
+
+  resource "xineramaproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xineramaproto-1.2.1.tar.bz2"
+    sha256   "977574bb3dc192ecd9c55f59f991ec1dff340be3e31392c95deff423da52485b"
+  end
+
+  resource "xproto" do
+    url      "http://ftp.x.org/pub/individual/proto/xproto-7.0.28.tar.bz2"
+    sha256   "29e85568d1f68ceef8a2c081dad9bc0e5500a53cfffde24b564dc43d46ddf6ca"
+  end
+
+  ##### END of XPROTOCOL HEADERS 
+
+  resource "xcb-proto" do
+    url "http://xcb.freedesktop.org/dist/xcb-proto-1.11.tar.gz"
+    sha256 "d12152193bd71aabbdbb97b029717ae6d5d0477ab239614e3d6193cc0385d906"
+  end
+
+## for libxcb... not sure where it is needed there
```

Comment:
Strictly speaking, libpthread-stubs isn't necessary on Linux. [BLFS](http://www.linuxfromscratch.org/blfs/view/svn/x/libxcb.html) actually recommends stripping the dependency on it out:

> sed "s/pthread-stubs//" -i configure: This sed removes dependency on libpthread-stubs package which is useless on Linux. 

I wasn't wholly comfortable with that, so I decided to just include it as a resource.